modules/monitoring-granting-users-permission-to-configure-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+[id="granting-users-permission-to-configure-alert-routing-for-user-defined-projects_{context}"]
+= Granting users permission to configure alert routing for user-defined projects
+
+You can grant users permission to configure alert routing for user-defined projects.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* The user account that you are assigning the role to already exists.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* Assign the `alert-routing-edit` role to a user in the `openshift-user-workload-monitoring` project:
```

Comment:
Assign the `alert-routing-edit` role to a user in the `user defined` project:

Reply:
Fixed